stdlib/collections/__init__.pyi

```diff
@@ -247,23 +238,23 @@ class Counter(Dict[_T, int], Generic[_T]):
     def __iand__(self, other: Counter[_T]) -> Counter[_T]: ...
     def __ior__(self, other: Counter[_T]) -> Counter[_T]: ...  # type: ignore
 
-class _OrderedDictKeysView(KeysView[_KT], Reversible[_KT]):
-    def __reversed__(self) -> Iterator[_KT]: ...
+class _OrderedDictKeysView(_dict_keys[_KT_co, _VT_co], Reversible[_KT_co]):
+    def __reversed__(self) -> Iterator[_KT_co]: ...
 
-class _OrderedDictItemsView(ItemsView[_KT, _VT], Reversible[Tuple[_KT, _VT]]):
-    def __reversed__(self) -> Iterator[Tuple[_KT, _VT]]: ...
+class _OrderedDictItemsView(_dict_items[_KT_co, _VT_co], Reversible[Tuple[_KT_co, _VT_co]]):
+    def __reversed__(self) -> Iterator[Tuple[_KT_co, _VT_co]]: ...
 
-class _OrderedDictValuesView(ValuesView[_VT], Reversible[_VT]):
-    def __reversed__(self) -> Iterator[_VT]: ...
+class _OrderedDictValuesView(_dict_values[_VT_co, _KT_co], Reversible[_VT_co], Generic[_KT_co, _VT_co]):
```

Comment:
Can you switch the Generic order here too for consistency? I guess it comes up less often because OrderedDicts aren't used as frequently, but I'd assume the same bug as with dict would appear otherwise.